src/membrane-keypad-element.ts

```diff
@@ -182,4 +187,26 @@ export class MembraneKeypadElement extends LitElement {
       );
     }
   }
+
+  private keyStrokeDown(key: string) {
+    const keyName = `key-${key.toLowerCase()}`;
+    const text = keyName[keyName.length - 1];
+    const selectedKey = this.shadowRoot?.querySelector(`[data-key-name="${keyName}"]`);
+    this.down(text, selectedKey as SVGElement);
+  }
+
+  private keyStrokeUp(key: string) {
+    let selectedKey = this.shadowRoot?.querySelector('.pressed');
+    const keyName = `key-${key.toLowerCase()}`;
+
+    if (key === 'Shift') {
+      this.shiftPressed = false;
+      selectedKey?.classList.remove('pressed');
```

Comment:
Actually, this means that if I click on `1`, then press `Shift` and leave it, it will seem like `1` is no longer pressed, even though the actual release event will only be fired when I release the `1` key (which is the expected behavior)

Reply:
That's right. I tracked the difference between the numbers and the signs with this variable (8 vs * and 3 vs #). Now I use e.key and it is no longer needed. But I have a new problem now. Without this part, if I press shift+8 -> the * button seems pressed. Then if I release the shift before the 8, I get a press and release of 8 and the * is still shown as pressed. That also happens in the opposite way of pressing 8, then press shift and release 8 before the shift. The 8 still has the pressed class. The only way I can think of is checking the pressedKeys var: check for each case of 3,8,# and * with shift on keyStrokeDown and keyStrokeUp. Is this solution too hard-coded in your opinion?